create_jenkins_job.py

```diff
@@ -188,6 +188,18 @@ def create_job(os_name, job_name, template_file, additional_dict):
                 'mailer_recipients': DEFAULT_MAIL_RECIPIENTS,
             })
 
+        # configure nightly triggered job using opensplice
+        job_name = 'nightly_' + job_os_name + '_ospl' + '_release'
+        if os_name == 'windows':
+            job_name = job_name[:20]
+        create_job(os_name, job_name, 'ci_job.xml.em', {
+            'cmake_build_type': 'Release',
+            'time_trigger_spec': PERIODIC_JOB_SPEC,
+            'mailer_recipients': DEFAULT_MAIL_RECIPIENTS,
+            'use_connext_default': 'false',
+            'use_opensplice_default': 'true',
```

Comment:
if we want fastrtps to be used I think it's better to set its checkbox explicitly to true also

Reply:
I originally didn't feel like doing it as the goal here was just to add testing for opensplice regardless of our default RMW.
As I needed to disable connext explicitly, it cannot hurt to also be explicit about using fastrtps even if it's on for all jobs by default. https://github.com/ros2/ci/pull/176/commits/ba02f44828aaa4d2b87b7c8b6dbc542d977571ac